src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -19,31 +19,24 @@
 
 package net.snowflake.spark.snowflake
 
-import java.sql.{
-  Connection,
-  DriverManager,
-  PreparedStatement,
-  ResultSet,
-  ResultSetMetaData,
-  SQLException,
-  Statement
-}
+import java.sql.{Connection, DriverManager, PreparedStatement, ResultSet, ResultSetMetaData, SQLException, Statement}
 import java.util.Properties
 import java.util.concurrent.atomic.AtomicInteger
 import java.util.concurrent.{Executors, ThreadFactory}
 
 import net.snowflake.client.jdbc.telemetry.{Telemetry, TelemetryClient}
-
-import scala.concurrent.{Await, ExecutionContext, Future}
-import scala.concurrent.duration.Duration
-import scala.util.Try
-import org.apache.spark.sql.execution.datasources.jdbc.DriverRegistry
-import org.apache.spark.sql.types._
-import org.slf4j.LoggerFactory
+import net.snowflake.spark.snowflake.DefaultJDBCWrapper.DataBaseOperations
```

Comment:
In general, I suggest not to change the "import" exception necessary.

Reply:
scalafmt formatter changed these. :(